vhdbuilder/scripts/cni-prefetch-driver.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+CNI_PREFETCH_SCRIPT_PATH="/opt/azure/containers/prefetch.sh"
+
+if [ ! -f "$CNI_PREFETCH_SCRIPT_PATH" ]; then
+    echo "CNI prefetch script does not exist at $CNI_PREFETCH_SCRIPT_PATH"
+    return 1
+fi
+
+chmod +x $CNI_PREFETCH_SCRIPT_PATH
+echo "running CNI prefetch driver script at $CNI_PREFETCH_SCRIPT_PATH..."
+sudo /bin/bash $CNI_PREFETCH_SCRIPT_PATH
+echo "CNI prefetch driver script completed successfully"
+
+echo "deleting CNI prefetch driver script at $CNI_PREFETCH_SCRIPT_PATH..."
+rm -- "$0"
```

Comment:
why not just `rm -f $CNI_PREFETCH_SCRIPT_PATH`?

Reply:
rm -- "$0" deletes itself, so thats just what I went with. I changed it to rm -f $CNI_PREFETCH_SCRIPT_PATH for clarity :)

Reply:
rm -- "$0" deletes my file made in this PR while rm -f $CNI_PREFETCH_SCRIPT_PATH removes your file that the script is running. Which one do we wnat deleted?